src/audiomixerboard.cpp

```diff
@@ -1210,92 +1210,98 @@ void CAudioMixerBoard::ApplyNewConClientList ( CVector<CChannelInfo>& vecChanInf
-                        // adjusted. If we have received our own channel ID, then
-                        // we can adjust the level even if no fader was visible.
-                        // The fader level of 100 % is the default in the
-                        // server, in that case we do not have to do anything here.
-                        if ( ( !bNoFaderVisible || ( ( iMyChannelID != INVALID_INDEX ) && ( iMyChannelID != i ) ) ) &&
-                             ( pSettings->iNewClientFaderLevel != 100 ) )
-                        {
-                            // the value is in percent -> convert range
-                            vecpChanFader[i]->SetFaderLevel ( pSettings->iNewClientFaderLevel / 100.0 * AUD_MIX_FADER_MAX );
-                        }
-                    }
-
-                    // restore gain (if new name is different from the current one)
-                    if ( vecpChanFader[i]->GetReceivedName().compare ( vecChanInfo[j].strName ) )
-                    {
-                        // the text has actually changed, search in the list of
-                        // stored settings if we have a matching entry
-                        int  iStoredFaderLevel;
-                        int  iStoredPanValue;
-                        bool bStoredFaderIsSolo;
-                        bool bStoredFaderIsMute;
-                        int  iGroupID;
-
-                        if ( GetStoredFaderSettings ( vecChanInfo[j].strName,
-                                                      iStoredFaderLevel,
-                                                      iStoredPanValue,
-                                                      bStoredFaderIsSolo,
-                                                      bStoredFaderIsMute,
-                                                      iGroupID ) )
-                        {
-                            vecpChanFader[i]->SetFaderLevel ( iStoredFaderLevel, true ); // suppress group update
-                            vecpChanFader[i]->SetPanValue ( iStoredPanValue );
-                            vecpChanFader[i]->SetFaderIsSolo ( bStoredFaderIsSolo );
-                            vecpChanFader[i]->SetFaderIsMute ( bStoredFaderIsMute );
-                            vecpChanFader[i]->SetGroupID ( iGroupID ); // Must be the last to be set in the fader!
-                        }
-                    }
-
-                    // set the channel infos
-                    vecpChanFader[i]->SetChannelInfos ( vecChanInfo[j] );
-
-                    bFaderIsUsed = true;
+                    // the value is in percent -> convert range
+                    vecpChanFader[iChanID]->SetFaderLevel ( pSettings->iNewClientFaderLevel / 100.0 * AUD_MIX_FADER_MAX );
                 }
             }
 
-            // if current fader is not used, hide it
-            if ( !bFaderIsUsed )
+            // restore gain (if new name is different from the current one)
```

Comment:
This, again, is a duplicate comment -- the comment should be inside the if describing the state there.  There's already a comment which does that in the right place.